log_upload.py

```diff
@@ -0,0 +1,222 @@
+"""
+ASSUMPTIONS:
+    log file is in the SAME directory as this script
```

Comment:
This isn't true - in `PATHWAY.md`, you state that the "new logfile is created inside the `logfiles` folder".